test-case-reporting/.gitignore

```diff
@@ -2,3 +2,4 @@ dist/
 *.js
 !.eslintrc.js
 *.env
+!redacted.env
```

Comment:
Nit: Instead of explicitly allowing `redacted.env`, let's just explicitly disallow `.env`

Reply:
Done.

If I'm understanding correctly, this makes it so it won't track `redacted.env` at all anymore. Is that what we want?

Reply:
See suggestion below